_docker/drupal/db-check.php

```diff
@@ -10,8 +10,8 @@
  */
 try {
     $DBH = new PDO(
-        "mysql:host=drupalmysql;dbname=$_ENV[DB_NAME]",
```

Comment:
Same as comments below: We'll need to be able to parameterise the db username/password/host details. My initial thought is that its easiest to move this functionality into your install Ruby script so that the loading of configuration is only done once. Before running the install process, you could check/wait on the database to become available.